srcpkgs/bro/template

```diff
@@ -1,49 +1,11 @@
 # Template file for 'bro'
 pkgname=bro
 version=2.6.4
-revision=1
-archs="x86_64* i686* aarch64* armv7* ppc64*"
-build_style=cmake
-hostmakedepends="flex pkg-config python3"
-makedepends="bind-devel geoip-devel libpcap-devel libressl-devel jemalloc-devel"
-short_desc="Advanced framework for network traffic analysis"
+revision=2
+archs=noarch
+build_style=meta
+depends="zeek"
+short_desc="Advanced framework for network traffic analysis (transitional dummy package)"
 maintainer="Andrew Benson <abenson+void@gmail.com>"
 license="BSD-3-Clause"
```

Comment:
What would be the correct license to put here?

Reply:
[Manual.md](https://github.com/void-linux/void-packages/blob/7c361a68838e462bea726d95534de877c1426460/Manual.md) states: 
```
Empty meta-packages that don't include any files which thus have and require no license, should have set license="BSD-2-Clause".
```